### DISCUSSION TITLE
CubeSatSim 1.3

### DISCUSSION THREAD

**Discussion by N2OA** [alanbjohnston/CubeSatSim#292] [General]
I'm interesting in building a few of the new CubeSatSim's. Any news on when the boards will be available? Maybe a more important question, is there an alternate supplier of the radio module? The supplier listed is taking a break. https://www.tindie.com/products/sunrise/sr_0w5u-two-way-radio-module05w400m-480m/

Dave N2OA

**Comment by alanbjohnston** [comment]
Hi Dave,

The v1.3.2 Beta is very close to being available.  I have updated the alias https://CubeSatSim.org/bom-beta to point to the latest BOM.  in the SMT Parts tab you will see an alternative supplier.  

73,
Alan
KU2y

**Reply by Boovtech** [reply]
Hi Alan,

This is a little off the subject.  But it looks like the options for SMT vs. through-hole board are in reverse on the BOM.  The one that says WITHOUT SMT is priced $49 but is grouped with the list of parts and is marked "various" for suppliers.  While the one marked WITH SMT is listed at $10 and marked with "AMSAT" as the supplier.  It seems opposite of what it should be.

Image attached.

Virginia NV5F

 
![BOM_SMT](https://github.com/alanbjohnston/CubeSatSim/assets/108498218/32e701fe-b595-43a9-92f6-963ca46bb424)

**Comment by N2OA** [comment]
Thank you Alan. I replied to your email a few days ago. Just wanted to make sure it didn't end up in the spam folder.

Dave N2OA
